src/main/java/cd/rr/calculator/Calculator.java

```diff
@@ -1,7 +1,82 @@
 package cd.rr.calculator;
 
+import java.util.Scanner;
+
 public class Calculator {
     public static void main(String[] args) {
         // Your solution here!
+           int i=0;
+            while ( i < 1) {
```

Comment:
I would recommend either using a flag here i.e. `while(running)` where running is set to true